app/models/user.rb

```diff
@@ -222,36 +222,13 @@ def pending_notifications_count
   def assign_default_role
     @just_created = true
     add_role(:submitter, default_group) unless has_role?(:submitter, default_group)
-    enable_messages_from(group: default_group)
+    default_group.enable_messages_for(user: self)
   end
 
   # Returns true if the user has notification e-mails enabled
   # @return [Boolean]
   def email_messages_enabled?
     email_messages_enabled
   end
-
```

Comment:
We have the logic duplicated, which I only found out because the tests failed.  Since User is so large I though utilizing the code in Group made sense.